issue-onboarding/onboarding-event-listener.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: EventListener
+metadata:
+  name: onboarding-issue-el
+spec:
+  serviceAccountName: onboarding-issue-reciever-sa
+  triggers:
+    - name: github-issue-release
+      interceptors:
+        - github:
+            secretRef:
+              secretName: github-secret
+              secretKey: webhooksecret
+            eventTypes:
+              - issues
+        - cel:
+            filter: "(body.issue.title.startsWith('NEW PROJECT')) && (body.action in ['opened'])"
+      bindings:
+        - kind: TriggerBinding
+          ref: onboarding-issue-binding
+      template:
+        ref: onboarding-issue-template
```

Comment:
Add this part to the eventlistener: https://github.com/AICoE/aicoe-ci/blob/master/trigger/eventlistener.yaml